### ISSUE TITLE
How run Patiobar on startup

### ISSUE THREAD

**Issue by thisdavej** [kylejohnson/Patiobar#3]
Hi Kyle - awesome work on Patiobar! I was going to create my own Node web server interface to pianobar and then discovered your excellent work!

I have a question.  I'm seeking to start up Patiobar on boot so I can play music on my RPi3 headlessly in the living room. Can't seem to figure out how to make it start on boot. Trying to add to `/etc/rc.local`.

Here's my startup script (/home/pi/Patiobar/startup.sh):
```bash
#!/bin/bash
screen -S pianobar -d -m pianobar

# experimented with this too
# sudo -iu pi /usr/bin/screen -S pianobar -d -m bash -c 'pianobar; exec bash'

node index.js
```
It works great if I boot and then run `startup.sh` from the terminal.  There may be some issues with using `screen` on startup.

I could possibly host a simple `netcat` server to receive a command via UDP to start it up, but it would be great if it would start on boot.  I could then have dedicated microSD card as a Patiobar music system.

**Comment by thisdavej** [comment]
After a number of additional iterations, I was able to get it working.  I created a file called `/home/pi/Patiobar/startup.sh` with the following contents:
```bash
#!/bin/bash

# Sleep for a few seconds to enable network to stabilize
sleep 15

screen -S pianobar -d -m bash -c 'pianobar'

cd /home/pi/Patiobar
node index.js
```
... and added a line near the bottom of `/etc/rc.local` (just above `exit 0`) like this:

```bash
sudo -iu pi /home/pi/Patiobar/startup.sh &

# this next line was already there
exit 0
```
Thanks!

**Comment by bt1957** [comment]
Hello Kyle...

I have your Patiobar running on top of pianobar, and I am really really happy with this setup.
Installed on a new Ras Pi 3 that I dedicated to the "Remote Pandora" task and I had a number of items that were not already installed that caused problems,  The main culprit was npm needed to be installed.  Once I got past that, I used npm to install the items that it calls in the index.js.

Thank you for a wonderful program!  Also thanks to thisdavej for his startup instructions, have that working wonderfully, too.

Happy Camper in Colorado Springs!

Cheers!
